libs/island-ui/core/src/lib/RadioButton/RadioButton.tsx

```diff
@@ -73,7 +74,7 @@ export const RadioButton = ({
         >
           <div className={styles.checkMark} />
         </div>
-        {label}
+        {HtmlParser(label)}
```

Comment:
I'm all for flexibility, and maybe I'm not that familiar with the use cases, but aren't we being to generous here with a custom label? if the label needs to be other than string wouldn't it be better to change the label type to something like `label?:  ReactNode` and handle the parsing outside the component if needed?

Reply:
side note: `string` should be valid as a `ReactNode`

Reply:
ofc.! Good comment
Will change that!